rgw/v2/tests/s3_swift/test_swift_bulk_delete.py

```diff
@@ -0,0 +1,168 @@
+        container_name(str): Container name
+        user_id(str): User ID
+        oc(int): object count
+        cc(int): container count
+        size(int): Object size
+    """
+    swift_object_name = utils.gen_s3_object_name('%s.container.%s' % (user_id, cc), oc)
+    log.info('object name: %s' % swift_object_name)
+    object_path = os.path.join(TEST_DATA_PATH, swift_object_name)
+    log.info('object path: %s' % object_path)
+    data_info = manage_data.io_generator(object_path, size)
+    # upload object
+    if data_info is False:
+        raise TestExecError("data creation failed")
+    log.info('uploading object: %s' % object_path)
+    with open(object_path, 'r') as fp:
+        rgw.put_object(container_name, swift_object_name,
+                       contents=fp.read(),
+                       content_type='text/plain')
+    return swift_object_name
+
+def test_exec(config):
+    """
+    Executes test based on configuration passed
+    Args:
+        config(object): Test configuration
+    """
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    umgmt = UserMgmt()
+    ceph_conf = CephConfOp()
+    rgw_service = RGWService()
+    # preparing data
+    user_names = ['tuffy', 'scooby', 'max']
+    tenant = 'tenant'
+    tenant_user_info = umgmt.create_tenant_user(tenant_name=tenant, user_id=user_names[0],
+                                                displayname=user_names[0])
+    user_info = umgmt.create_subuser(tenant_name=tenant, user_id=user_names[0])
+    auth = Auth(user_info)
+    rgw = auth.do_auth()
+
+    container_name = utils.gen_bucket_name_from_userid(user_info['user_id'], rand_no=0)
+    container = swiftlib.resource_op({'obj': rgw,
+                                      'resource': 'put_container',
+                                      'args': [container_name]})
+    if container is False:
+        raise TestExecError("Resource execution failed: container creation faield")
+    for oc, size in list(config.mapped_sizes.items()):
+        # upload objects to the container
+        swift_object_name = fill_container(rgw, container_name, user_names[0], oc, 0, size)
+        # delete all uploaded objects
+        log.info('deleting all swift objects')
+    auth_response = rgw.get_auth()
+    token = auth_response[1]
+    # test.txt file should contain container_name
+    with open("test.txt", "w") as f:
+        f.write(container_name)
+    ip_and_port = rgw.authurl.split('/')[2]
+    url = 'http://{}/swift/v1/?bulk-delete'.format(ip_and_port)
+    test_file = open("test.txt", "r")
+    
+    headers = {"Accept": "application/json", "Content-Type": "text/plain",
+               "X-Auth-Token": token}
+    response = requests.delete(url, headers=headers,
+                               files={"form_field_name": test_file})
+    if response.status_code == 200:
+        print('Bulk delete succeeded')
```

Comment:
can you remove the `prints` here ? and below too

Reply:
@rakeshgm, I have given print statement to mention test is passed. Can I give log.info instead of print?

Reply:
yes you can use log